lambda_code/logs/ResourcePolicy/index.py

```diff
@@ -0,0 +1,33 @@
+"""
+Custom resource to support Amazon Cloudwatch Logs ResourcePolicy.
+https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_ResourcePolicy.html
+"""
+
+from cfn_custom_resource import CloudFormationCustomResource
+
+
+class ResourcePolicy(CloudFormationCustomResource):
```

Comment:
Mogelijk wil je RESOURCE_TYPE_SPEC ook zetten, anders verlies je de `logs` in je naam. [1] [2]

[1]: https://github.com/iRobotCorporation/cfn-custom-resource/wiki/Basic-usage
[2]: https://github.com/vrtdev/custom-resources/blob/master/lambda_code/autoscaling/Renotify/index.py#L8